airflow/providers/mongo/hooks/mongo.py

```diff
@@ -40,11 +40,12 @@ class MongoHook(BaseHook):
     ex.
         {"srv": true, "replicaSet": "test", "ssl": true, "connectTimeoutMS": 30000}
 
-    :param conn_id: This is the Airflow connection to use for Mongo credentials.
-    :type conn_id: str
+    :param mongo_conn_id: The :ref:`Mongo connection id <howto/connection:mongo>` to use
+        when connecting to MongoDB.
+    :type mongo: str
     """
 
-    conn_name_attr = 'conn_id'
+    conn_name_attr = 'mongo_conn_id'
```

Comment:
don't think this will work as the kwarg on L53 needs changing to `mongo_conn_id` from `conn_id`.

Lets remove this change from this PR

Reply:
@kaxil just fixed this and rebased to apache/master